### PR TITLE
[ACR-564][NUI] deprecate unused API

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
deprecate unused API.
ElmSharp, ElmSharp.Wearable and NUI.Wearable.
these whole modules are being deprecated.

### API Changes ###
https://jira.sec.samsung.net/browse/TCSACR-564

### CODE REVIEW

File: src/Tizen.NUI.Wearable/src/public/NUIWatchApplication.cs [hunk 1]
```diff
@@ -25,26 +25,31 @@ namespace Tizen.NUI
...
+        [Obsolete("This has been deprecated in API12")]
         public event EventHandler TimeTick;
 
         /// <summary>
         /// Event arguments that passed via time tick event signal.
         /// </summary>
+        [Obsolete("This has been deprecated in API12")]
         public class TimeTickEventArgs : EventArgs
         {
             private WatchTime _watchTime;
```
Comment (wiertel) [inline src/Tizen.NUI.Wearable/src/public/NUIWatchApplication.cs:55]: consider marking those 2 public properties as Obsolete as well, the class is already deprecated, but for consistency members might be as well

File: src/Tizen.NUI.Wearable/src/public/RecyclerView/RecyclerView.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ namespace Tizen.NUI.Wearable
     /// <summary>
     /// [Draft] This class provides a View that can recycle items to improve performance.
     /// </summary>
-    /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+    [Obsolete("This has been deprecated in API12")]
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class RecyclerView : ScrollableBase
     {
```
Comment (wiertel) [inline src/Tizen.NUI.Wearable/src/public/RecyclerView/RecyclerView.cs:30]: consider marking this first constructor as well

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/ElmSharp/ElmSharp/EvasImage.cs [hunk 3]
```diff
@@ -290,6 +307,7 @@ public void SetNativeSurface(IntPtr surface)
         /// <param name="top">The border's top width.</param>
         /// <param name="bottom">The border's bottom width.</param>
         /// <since_tizen> preview </since_tizen>
+        [Obsolete("This has been deprecated in API12")]
         public void SetBorder(int left, int right, int top, int bottom)
         {
             Interop.Evas.evas_object_image_border_set(RealHandle, left, right, top, bottom);
```
[the remark this answers is not in the document]
Comment (dongsug-song) [reply src/ElmSharp/ElmSharp/EvasImage.cs:313]: Thank you for review. 
it has been fixed.

File: src/ElmSharp/ElmSharp/EvasObject.cs [hunk 4]
```diff
@@ -833,6 +876,7 @@ public bool AllEventsFrozen
         /// Sets or gets the layer of its canvas that the given object will be part of.
         /// </summary>
         /// <since_tizen> preview </since_tizen>
+        [Obsolete("This has been deprecated in API12")]
         public virtual int Layer
         {
             get
```
[the remark this answers is not in the document]
Comment (dongsug-song) [reply src/ElmSharp/ElmSharp/EvasObject.cs:883]: Thank you for review. 
it has been fixed.

File: src/Tizen.NUI.Wearable/src/public/NUIWatchApplication.cs (hunk 1 above)
Comment (dongsug-song) [reply src/Tizen.NUI.Wearable/src/public/NUIWatchApplication.cs:55]: Thank you for review. 
it has been fixed.

File: src/Tizen.NUI.Wearable/src/public/RecyclerView/RecyclerView.cs (hunk 2 above)
Comment (dongsug-song) [reply src/Tizen.NUI.Wearable/src/public/RecyclerView/RecyclerView.cs:30]: Thank you for review.
it has been fixed.

**Review by wiertel — APPROVED** [review]

**Pull request merged**